README.md

```diff
@@ -1,3 +1,57 @@
-# taoPublishing
+# TAO _taoPublishing_ extension
 
-Pushes a test to a delivery environment
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+![GitHub](https://img.shields.io/github/license/oat-sa/extension-tao-publishing.svg)
+![GitHub release](https://img.shields.io/github/release/oat-sa/extension-tao-publishing.svg)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/oat-sa/extension-tao-publishing.svg)
+
+> Synchronize deliveries, tests and interactions to other tao platforms during the delivery creation.
+
+
+## Installation instructions
+
+These instructions assume that you have already a TAO installation on your system. If you don't, go to
+[package/tao](https://github.com/oat-sa/package-tao) and follow the installation instructions.
+
+
+Add the extension to your TAO composer and to the autoloader:
+```bash
+$ composer require oat-sa/extension-tao-publishing ^2.1 --prefer-source
+```
+
+Install the extension on the CLI from the project root:
+
+**Linux:**
+```bash
+sudo php tao/scripts/installExtension.php taoPublishing
+```
+
+**Windows:**
+```bash
+php tao\scripts\installExtension.php taoPublishing
+```
+
+**Docker**
+```bash
+$ docker container exec -it {YOUR-TAO-INSTANCE-PHP-CONTAINER} php tao/scripts/installExtension.php taoPublishing
+```
+
+After the install just log out and log in again on the TAO Platform.
+
+As a system administrator you can also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoPublishing_ on the right hand side, check the box and hit _install_
+
+## Set up deliveries on TAO Platform
+
+To be able to synchronize, you need to set up the platforms that you want to sync. You can do it by:
```

Comment:
What about `Publish Actions`. As I remember this is the list of actions for triggering publishing to a remote env

Reply:
I forgot to mention it. I'll do that soon. Many thanks.

Reply:
Hi @tikhanovichA. I checked in the code by removing the config on the 
config/taoPublishing/PublishingService.conf.php and config/default/PublishingService.conf.php and the behavior that I got was that the extension just removes the event from the admin platform configuration but doesn't` mean that the sync action will be removed. For example, if you remove the DeliveryCreatedEvent from the config, it'll remove it from the admin platform form, but the user can still sync it when creating a new delivery.
So, in this case, this is not something that we can set up actually. That's why I wouldn't put anything related to this on the readme.

Reply:
it looks like a bug.
So, if I will not use that block with `Publish Actions`, how taoPublishing will work?

Reply:
Hello @tikhanovichA. After more investigation, I could find more details about how to set up the Publish Actions. Could you please take a look at the README changes again, please?